src/pytest_inmanta_lsm/plugin.py

```diff
@@ -132,10 +135,40 @@ def remote_orchestrator_container(
         yield None
         return
 
+    orchestrator_image = inm_lsm_ctr_image.resolve(request.config)
+    latest_compose_file = pathlib.Path(__file__).parent / "resources/docker-compose.yml"
+    legacy_compose_file = pathlib.Path(__file__).parent / "resources/docker-compose-iso6.yml"
+    try:
+        compose_file = inm_lsm_ctr_compose.resolve(request.config)
+    except pytest_inmanta.test_parameter.ParameterNotSetException:
+        # The compose file is not set, we can then either use the default one, or
+        # the default legacy one (for <iso7.1).  To decide which one is the most
+        # appropriate, we parse the container image tag and extract the iso version
+        iso_major_version_match = re.fullmatch(
+            r"(?:.*)\/service-orchestrator:((\d+(?:\.\d+)*)(?:\-dev)?|dev)",
+            orchestrator_image,
+        )
+        if not iso_major_version_match:
+            LOGGER.warning(
+                "Can not parse orchestrator image tag: %s.  Using legacy docker-compose file",
+                orchestrator_image,
+            )
+            compose_file = legacy_compose_file
+        elif iso_major_version_match.group(1) == "dev":
+            # Latest dev, use the the latest compose file
```

Comment:
```suggestion
            # Latest dev, use the the latest compose file, this is always safe because we don't distribute this externally
```

This is not too relevant now, but if we ever want to do the same for iso 7 vs 8, this becomes important, because not everyone has access to the same set of images, so concepts as "latest" are ambiguous. But as long as we only use such a concept for "latest *dev*", we're good because that concept only exists internally, hence everyone (who might use it) does have the same set of images available.